cmd/xl-storage-disk-id-check.go

```diff
@@ -687,6 +687,9 @@ func (p *xlStorageDiskIDCheck) updateStorageMetrics(s storageMetric, paths ...st
 			if errors.Is(err, context.DeadlineExceeded) || errors.Is(err, context.Canceled) {
 				atomic.AddUint64(&p.totalErrsTimeout, 1)
 			}
+			if (err.Error() == context.DeadlineExceeded.Error()) || (err.Error() == context.Canceled.Error()) {
```

Comment:
We will be double counting the metrics here

Reply:
yes testing code, now removed @vadmeste